core/conversion/converters/impl/element_wise.cpp

```diff
@@ -144,6 +144,41 @@ auto element_wise_registrations TRTORCH_UNUSED =
                     LOG_DEBUG("Output tensor shape: " << out->getDimensions());
                     return true;
                   }})
+        .pattern({"aten::clamp(Tensor self, Scalar? min=None, Scalar? max=None) -> (Tensor)",
+                  [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+                    // Compute min(max(min_threshold, input), max_threshold)
+                    auto self = args[0].ITensorOrFreeze(ctx);
+                    auto clamp_layer_out = self;
+                    if (args[1].isIValue() && args[1].IValue()->isScalar()) {
```

Comment:
I think we might be able to do !isNone on the IValue instead of two checks

Reply:
If I remember correctly, isNone() didn't work for me. I will double check again.

Reply:
The op does basically this `min(max(min_threshold, input), max_threshold)`
In the case when min_threshold or max_threshold is not given, args[1] or args[2] will be registered as an IValue. 
The isNone() check will check if the type is one of { kITensor, kIValue, kNone }. The type in this case would be IValue so isNone() will not return true.

Reply:
I see